src/main/markdown/release-notes.md

```diff
@@ -59,10 +59,79 @@ The GWT Release Notes
+- Source level set to Java 8.
+- Static and default methods in interfaces aren't visible to generators. If you want to take advantage of those Java-8isms, you're encouraged to switch to an annotation processor. This could break existing build if an interface is changed to turn a non-default method into a default method.
 
 ### Deprecations
-- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'. -->
+- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'.
+- Deprecate JSR 303 Bean Validation support (javax.validation, c.g.g.validation). The code hasn't been maintained for years.
+
+### Breaking changes
+- GIN needs a workaround now: gwtproject/gwt#9311
+- JSNI disallows method references to Double, String, Boolean and its super types: gwtproject/gwt#9356
+
+### Compiler
+- Update JS reserved keywords to ES6.
+- Fix instanceof for primitives & string.
+- Make exceptions work across Java/JavaScript boundary
+- Allow casting native arrays to Object[].
+- Use patched JDT to circumvent memory leak when compiling for Java 8.
+
+### JsInterop
+- Implement @JsOverlay with default methods in Java8.
+- Split JsInterop annotations into their own JAR when deploying to Maven:
+      <dependency>
+          <groupId>com.google.jsinterop</groupId>
+          <artifactId>jsinterop-annotations</artifactId>
+      </dependency>
+- Disallow non-native @JsProperty on static methods.
+- Remove legacy JsInterop.
+- Add JsOptional annotation (disallowed on primitive typed parameters).
+- Allow @JsOverlay on effectively final methods.
+
+### UIBinder
+- Add support for JsType class in UIBinder.
+
+### JDK emulation
+- Emulate java.io.UncheckedIOException.
+- Emulate Optional<T> and its int, long, double variants.
+- Emulate Objects.requireNonNull() with message Supplier.
+- Fix Math.min/max(float/double) emulation behavior.
+- Emulate Character.isBmpCodePoint.
+- Emulate CharSequence.chars().
+- Add java.lang.SecurityException.
+- Emulate Java 8 API of 
+  - java.util.Arrays, 
+  - java.util.ArrayDeque,
+  - java.math.BigInteger,
+  - java.util.BitSet,
+  - java.util.Comparator,
+  - java.util.function,
+  - java.util.Iterator,
+  - java.lang.Iterable,
+  - java.util.IntSummaryStatistics/LongSummaryStatistics/DoubleSummaryStatistics
+  - java.util.Collection/Lists/Queues, 
+  - java.util.Map, 
+  - java.util.logging.Logger, 
+  - java.util.PrimitiveIterator,
+  - java.util.Spliterator,
+  - java.util.stream,
+  - java.util.StringJoiner
+
+## GSS
+- Add CSS3 and GSS support in InterfaceGenerator
+
+### Miscellanous
+- Fixing up the Maven samples to work with GWT 2.8
+- Fixed spelling error when starting CodeServer
+- Update globals for Chrome 52.0.2743.24.
+- Update apache commons-collections to 3.2.2.
+- JsonValue is now serializable. This enables storing JsonValues in the HTTP session on the server.
```

Comment:
Maybe be explicit that we're talking about `elemental.json.JsonValue`, not `c.g.g.json.client.JSONValue`